examples/tutorials/Introduction_To_Fourier_Neural_Operator.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>Can you add a citation for this method? I haven't seen this blurring method before</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4526/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>